docs/tutorial/sql.md

```diff
@@ -1094,8 +1094,8 @@ SedonaPyDeck.create_choropleth_map(df=groupedresult, plot_col='AirportCount')
 
 ![Creating a Choropleth map using SedonaPyDeck](../image/choropleth.gif)
 
-The dataset used is available [here](https://github.com/apache/sedona/tree/4c5fa8333b2c61850d5664b878df9493c7915066/binder/data/ne_50m_airports) and
-can also be found in the example notebook available [here](https://github.com/apache/sedona/blob/4c5fa8333b2c61850d5664b878df9493c7915066/binder/ApacheSedonaSQL_SpatialJoin_AirportsPerCountry.ipynb)
+The dataset used is available [here](https://github.com/apache/sedona/tree/master/docs/usecases/data/ne_50m_airports) and
```

Comment:
Please refer to the dataset via commit id like what it does right now. This will make sure we can always find the dataset.

Reply:
@jiayuasu Should I update all the linked I have change to use the latest available commit (both in `sql.md` and `ApacheSedonaSQL.ipynb`), Or only the `ne_50m_airports` dataset (the `ApacheSedonaSQL_SpatialJoin_AirportsPerCountry.ipynb` will be pointer through the `master` branch)?

Reply:
Only the dataset `ne_50m_airports` is necessary

Reply:
@jiayuasu got it, fixed